cypress/e2e/StackVisualizer.cy.ts

```diff
@@ -0,0 +1,51 @@
+import uaLocalization from "../../src/assets/i18n/ua.json";
+
+const STACK_TEST_MSG = "Test item";
+const PATH_NAME_STACK = "/stack";
+const STACK_LANG_MATCHER =
+  uaLocalization.translation.stackVisualizer.buttons.add;
+
+const inputTextAddNewStackItem = () => {
+  cy.getBySel(`data-list-text-input`).type(STACK_TEST_MSG);
+  cy.getBySel(`data-list-add-button`).click();
+  cy.getBySel(`data-list-items-container`).children().should("have.length", 4);
+  cy.contains(STACK_TEST_MSG);
+};
+
+const removeStackItem = () => {
+  cy.getBySel(`data-list-remove-button`).click();
+  cy.getBySel(`data-list-items-container`).children().should("have.length", 2);
```

Comment:
How you can be sure that your Item is removed? Also if we change markup and there will be 2 divs this test will pass as well.

Reply:
before every test issue we make cy.visit(${path_name}), so i always sure that the starting number of items in tessted list is == 3. But if i should check and compare to actual naumber of items in store i will do it.